### PR TITLE
add Redstone BridgedUSDC

### PR DESCRIPTION
**Pull request by alvrs**
**Description**

* Adding the Redstone deployment of `BridgedUSDC` (https://explorer.redstone.xyz/address/0xD5d59fC063e7548b6015A36fEb10B875924A19be?tab=txs)
* Adding Redstone to the list of Superchain chains

### CODE REVIEW

**Comment by nitaliano** [comment]
The repo isn't configured to support `redstone` yet this is why the validate action is failing.

These steps below should help get us started.
1. Add chain info here https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/src/chains.ts
2. Add the chain keys here https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/29a9605303f3cffba85dbb56f198ef045788dbd2/src/schemas.ts#L59-L85
3. Add the chain key here https://github.com/ethereum-optimism/ethereum-](https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/29a9605303f3cffba85dbb56f198ef045788dbd2/src/schemas.ts#L61-L87)optimism.github.io/blob/29a9605303f3cffba85dbb56f198ef045788dbd2/src/types.ts#L22-L45

**Comment by nitaliano** [comment]
when i run it locally it fails here https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/29a9605303f3cffba85dbb56f198ef045788dbd2/src/validate.ts#L231 the validation i believe runs against this abi https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/legacy/LegacyMintableERC20.sol

**Comment by alvrs** [comment]
oh interesting, the token is not using the `ILegacyMintableERC20` interface but the `IOptimismMintableERC20` interface and can be bridged via [`StandardBridge.bridgeERC20`](https://github.com/ethereum-optimism/optimism/blob/f940301caf531996eee4172e710b0decb7b78dde/packages/contracts-bedrock/src/universal/StandardBridge.sol#L199-L211). (The view function to get the bridge contract is `token.bridge()` in this case). Looking at the code it seems like i can provide a bridge override in the config? Let me try that.

File: src/chains.ts [hunk 1]
```diff
@@ -92,7 +100,7 @@ export const NETWORK_DATA: Record<Chain, Network> = {
       'https://rpc.sepolia-api.lisk.com'
     ),
     layer: 2,
-  },
+  }
```
Comment (alvrs) [inline src/chains.ts:103]:
```suggestion
  },
```

**Review by nitaliano — APPROVED** [review]

**Comment by nitaliano** [comment]
yea sorry about that we'll update the logic so we start checking `IOptimismMintableERC20` and fallback to checking the legacy interface

**Pull request merged**